target/linux/imx/image/cortexa7.mk

```diff
@@ -22,7 +22,9 @@ define Device/technexion_imx7d-pico-pi
   DEVICE_PACKAGES := kmod-sound-core kmod-sound-soc-imx kmod-sound-soc-imx-sgtl5000 \
 	kmod-can kmod-can-flexcan kmod-can-raw kmod-leds-gpio \
 	kmod-input-touchscreen-edt-ft5x06 kmod-usb-hid kmod-btsdio \
-	kmod-brcmfmac brcmfmac-firmware-4339-sdio cypress-nvram-4339-sdio
+	kmod-brcmfmac brcmfmac-firmware-4339-sdio cypress-nvram-4339-sdio \
+	kmod-mux-mmio kmod-video-imx7-csi kmod-video-imx-mipi-csis kmod-video-mux \
```

Comment:
The camera is optional, so the kernel modules should be optional as well.

Reply:
Acked.